docs/installation-and-operation/serialization.md

```diff
@@ -4,63 +4,126 @@ redirect_from:
   - /docs/latest/enterprise-guide/serialization
 ---
 
-## Serialization
+# Serialization
 
 {% include plans-blockquote.html feature="Serialization" self-hosted-only="true" %}
 
-Once you really get rolling with Metabase it's often the case that you'll have more than one instance of it spun up. You might have a couple of testing or development instances and a few production ones, or maybe you have a separate instance per office or region.
+Once you really get rolling with Metabase, it's often the case that you'll have more than one Metabase spun up. You might have a couple of testing or development instances and a few production ones, or maybe you have a separate instance per office or region.
 
-To help you out in situations like this, Metabase has a serialization feature which lets you create a snapshot, called a dump, of the contents of a Metabase instance that can then be loaded into another instance. This lets you do things like create a set of dashboards and charts in one Metabase instance and then easily copy those items to a number of other Metabase instances. You could also use this feature to enable a staging-to-production workflow for important dashboards or reports by dumping from a staging instance of Metabase and then loading that dump into your production instance(s). You can even put the dump files into version control and audit changes to them, as the YAML files contained within the dump are pretty readable.
+Serialization lets you do things like:
 
-If you're looking to do a one-time migration from H2 to MySQL/Postgres, then use the [migration guide instead](./migrating-from-h2.md).
+- **Snapshot your Metabase**. Serialization creates "dumps" of one Metabase, which you can load into another Metabase.
```

Comment:
I think it is worth mentioning that Serialization is **not** a backup mechanism. If they want to backup their instance, there are other ways of doing it.